tutorials/algorithms/10_pvqd.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>I think it would be nice to mention explicitly that you'll keep track of both the Hamiltonian and the magnetization (with the <code>aux_operators</code> argument of the <code>TimeEvolutionProblem</code>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>good point :+1: </p>